src/backend/src/services/reimbursement-requests.services.ts

```diff
@@ -64,6 +65,118 @@ export default class ReimbursementRequestService {
     return createdReimbursementRequest.reimbursementRequestId;
   }
 
+  /**
+   * sends the pending advisor reimbursements to the advisor
+   * @param sender the person sending the pending advisor list
+   * @param saboNumbers the sabo numbers of the reimbursement requests to send
+   */
+  static async sendPendingAdvisorList(
+    sender: User & {
+      teams: Team[];
+    },
+    saboNumbers: number[]
+  ) {
+    const financeTeam = await prisma.team.findUnique({
+      where: { teamId: process.env.FINANCE_TEAM_ID }
+    });
+
+    if (!financeTeam) throw new HttpException(500, 'Finance team does not exist!');
+
+    if (
+      !sender.teams.some((team) => team.teamId === process.env.FINANCE_TEAM_ID) &&
+      !(financeTeam.leaderId === sender.userId)
+    ) {
+      throw new AccessDeniedException(`You are not a member of the finance team!`);
+    }
+
+    const reimbursementRequests = await prisma.reimbursement_Request.findMany({
+      where: {
+        saboId: {
+          in: saboNumbers
+        }
+      }
+    });
+
+    if (reimbursementRequests.length < saboNumbers.length) {
```

Comment:
don't you need to check that reimbursementRequests isn't undefined before you do this?

Reply:
thats what this is checking